extensions/cornerstone/src/commandsModule.ts

```diff
@@ -1403,9 +1421,6 @@ function commandsModule({
     updateMeasurement: {
       commandFn: actions.updateMeasurement,
     },
-    clearMeasurements: {
-      commandFn: actions.clearMeasurements,
-    },
     jumpToMeasurement: {
       commandFn: actions.jumpToMeasurement,
```

Comment:
why `clearMeasurements` is removed, i see it is being used

Reply:
It is duplicated here and in the sr module.  I don't really care which one we leave, but there should only be one.